operations/vouchers.md

```diff
@@ -54,85 +57,87 @@ Returns all rate vouchers filtered by [Service](services.md#service), voucher co
+| `Extent` | [Voucher Extent](vouchers.md#voucher-extent) | required | Extent of data to be returned. Whether only specific voucher info should be returned or related items as well. |
 | `ServiceIds` | array of string | required, max 1000 items | Unique identifiers of [Services](services.md#service) where the vouchers belong to. |
 | `VoucherIds` | array of string | optional, max 1000 items | Unique identifiers of vouchers. |
+| `CompanyIds` | array of string | optional, max 1000 items | Unique identifiers of the companies. |
 | `VoucherCodeValues` | array of string | optional, max 1000 items | Value of voucher codes used by customers. |
-| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which [Voucher](#voucher) was updated. |
-| `ExternalIdentifiers` | array of string | optional, max 1000 items | Identifiers of [Voucher](#voucher) from external systems. |
-| `Extent` | [Voucher extent](#voucher-extent) | required | Extent of data to be returned. Whether only specific voucher info should be returned or related items as well. |
-| `ActivityStates` | array of string [Activity state](_objects.md#activity-state) | optional | Whether to return only active, only deleted or both records. |
+| `ActivityStates` | array of [Activity state](_objects.md#activity-state) | optional | Whether to return only active, only deleted, or both types of record. If not specified, both active and deleted records will be returned. |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months |  |
+| `ExternalIdentifiers` | array of string | optional, max 1000 items | Identifiers of [Voucher](vouchers.md#voucher) from external systems. |
 | `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned. |
 
-#### Voucher extent
+#### Voucher Extent
+Extent of data to be returned. Whether only specific voucher info should be returned or related items as well.
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Vouchers` | bool | optional | Whether the response should contain main information about vouchers. |
-| ~~`VoucherCodes`~~ | ~~bool~~ | ~~optional~~ | ~~Whether the response should contain voucher codes used by customers.~~ **Deprecated!** |
-| `VoucherAssignments` | bool | optional | Whether the response should contain assignments between vouchers and [Rates](rates.md#rate). |
-| ~~`Companies`~~ | ~~bool~~ | ~~optional~~ | ~~Whether the response should contain detail of related companies.~~ **Deprecated!** |
-| `Rates` | bool | optional | Whether the response should contain detail of assigned rates. |
+| `Vouchers` | boolean | required | Whether the response should contain main information about vouchers. |
```

Comment:
All of these properties were originally `optional`, but since its just `boolean` the `required `is correct imo

Reply:
They are optional by design, in case we'd add new extent, enforcing its presence would be a breaking change. But booleans behave like this, but it can be overriden in SwaggerSchema using `Nullable`:

```
[SwaggerSchema("... description ...", Nullable = true)]
```

Reply:
fixed [here](https://github.com/MewsSystems/gitbook-connector-api/pull/622/commits/e40a2b031110b46c50082de558da132314f62e27)